pom.xml

```diff
@@ -147,33 +146,19 @@
 
     <properties>
 
-
-        <hbase1.artifactId.suffix>hbase1.x</hbase1.artifactId.suffix>
-        <hbase2.artifactId.suffix>hbase2.x</hbase2.artifactId.suffix>
-
-        <shims1.artifactId>omid-hbase-shims-${hbase1.artifactId.suffix}</shims1.artifactId>
-        <shims2.artifactId>omid-hbase-shims-${hbase2.artifactId.suffix}</shims2.artifactId>
-
-        <!-- hbase-1 profile props are here and not in profile section to work better with intelij-->
-        <shims.artifactId>${shims1.artifactId}</shims.artifactId>
-        <java.version>1.7</java.version>
-        <hadoop.version>${hadoop1.version}</hadoop.version>
-        <hbase.version>${hbase1.version}</hbase.version>
-        <shims.module>hbase-1</shims.module>
-        <hbase.artifactId.suffix>${hbase1.artifactId.suffix}</hbase.artifactId.suffix>
+        <java.version>1.8</java.version>
+        <commons-io.version>2.11.0</commons-io.version>
 
         <!-- Basic properties -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- 3rd-Party Library Versioning -->
-        <hbase1.version>1.3.1</hbase1.version>
-        <hbase2.version>2.0.1</hbase2.version>
-        <hadoop1.version>2.7.5</hadoop1.version>
-        <hadoop2.version>3.0.0</hadoop2.version>
+        <hbase.version>2.4.12</hbase.version>
+        <hadoop.version>2.10.0</hadoop.version>
```

Comment:
We only support Hadoop 3 in Phoenix but only 2.10 in Omid? Seems like an odd inconsistency, esp since the old hadoop2.version here is actually 3.0.0

Reply:
We've never used hadoop2.version in the old build, it was just copy-paste remainder.

As Omid doesn't create shaded artifacts with Hbase bits, the Haodop we build with has no impact on the end product, only on what we run the tests with.

I'm tracking this in OMID-163, but as this would require that we rebuild HBase for Hadoop3 for testing the same way that we do for Phoenix, I have decided against doing this.

If you feel that this is important, I can work on OMID-163, but it's out of scope for this JIRA.